truss/remote/baseten/remote.py

```diff
@@ -0,0 +1,52 @@
+from truss.remote.baseten.auth import AuthService
+from truss.remote.baseten.core import (
+    archive_truss,
+    create_model,
+    exists_model,
+    upload_model,
+)
+from truss.remote.baseten.service import BasetenService
+from truss.remote.baseten.utils import base64_encoded_json_str
+from truss.remote.truss_remote import TrussRemote
+from truss.truss_handle import TrussHandle
+
+
+class BasetenRemote(TrussRemote):
+    def __init__(self, remote_url: str, auth_service: AuthService, **kwargs):
```

Comment:
it would be nice if we can keep this interface taking kwargs that can be stringified, so that we can just make it work easily with `truss push --remote baseten -- kwarg1=hi`

One way you can do that, is to take in an optional API key str and construct the AuthService in the constructor (or look for the API key in the env if not)